.pre-commit-config.yaml

```diff
@@ -28,15 +28,19 @@ repos:
       - id: flake8
         args: ["--max-line-length=120"]
         exclude: tests/conftest.py # ignore pytest fixtures
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v1.1.1" # Use the sha / tag you want to point at
+  - repo: local
     hooks:
       - id: mypy
-        args: [--no-strict-optional, --ignore-missing-imports]
+        files: \.py$
+        name: mypy
+        entry: mypy
+        language: python
+        args: ["--config-file=mypy.ini"]
         additional_dependencies:
           - "types-PyYAML"
           - "types-requests"
           - "sqlalchemy2-stubs"
           - "types-attrs"
           - "types-python-dateutil"
           - "types-mock"
+          - "types-tqdm"
```

Comment:
These 3 lines are not needed, since they are in dev dependencies, 4 lines above should move to dev dependencies as well